Sources/ComposableArchitecture/Documentation.docc/Extensions/ReducerProtocol.md

```diff
@@ -7,7 +7,7 @@
 - ``reduce(into:action:)-4nzr2``
 - ``State``
 - ``Action``
-- ``Effect``
+- ``EffectTask``
```

Comment:
Or Maybe ``EffectPublisher`` to directly land on the page with the meaningful documentation?

Reply:
I think so, for now:

```suggestion
- ``EffectPublisher``
```

We'll be able to clean this all up on the 1.0 release branch.